spec/system/login_spec.rb

```diff
@@ -41,13 +41,23 @@
     it "displays the right message when user's email has been marked as expired" do
       password = "myawesomepassword"
       user.update!(password:)
-      expired_user_password = Fabricate(:expired_user_password, user:, password:)
+      Fabricate(:expired_user_password, user:, password:)
 
       login_modal.open
       login_modal.fill(username: user.username, password:)
       login_modal.click_login
 
-      expect(login_modal).to have_content(I18n.t("login.password_expired"))
+      expect(login_modal.find("#modal-alert")).to have_content(
+        I18n.t("js.login.password_expired", reset_url: "/password-reset").gsub(/<.*?>/, ""),
+      )
+      login_modal.find("#modal-alert a").click
+      find("button.forgot-password-reset").click
+
+      wait_for(timeout: 5) { ActionMailer::Base.deliveries.count != 0 }
```

Comment:
I hope this won't flake out (there is another spec that waits for the same thing and it was causing problems)